keyboards/szrkbd/szr35/config.h

```diff
@@ -0,0 +1,17 @@
+// Copyright 2025 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define TAPPING_TOGGLE 2
```

Comment:
This option does not appear to be used anywhere. Should be left to the end user to configure anyway.
```suggestion
```